### PR TITLE
Add ScreenReaderText component 

### LINKED ISSUE #2273: Accessibility: Scan mode not functioning as expected on NVDA or Narrator
## Version

4.5.1

## Describe the bug

In Narrator and NVDA, users are unable to navigate through the transcript using AT navigation in scan mode. This makes it impossible to go through the transcript after the screen reader has read out the original received activities. 

## Expected behavior
Scan mode working as expected is minimal functionality necessary to be able to review the transcript.

## Additional context
Related to Ninja Camp :)

[Bug]

### PR DESCRIPTION
**Pull request by corinagum**
Fixes #2273 

## Changelog Entry

-  Fix [#2273](https://github.com/microsoft/BotFramework-WebChat/issues/2273). Add ScreenReaderText component, by [@corinagum](https://github.com/corinagum) in PR [#2278](https://github.com/microsoft/BotFramework-WebChat/pull/2278)

## Description

Narrator and NVDA were unable to examine the transcript of Web Chat in scan mode. These fixes re-enable scan mode with AT.

## Specific Changes

- Rename `<TimeAgo>` to `<RelativeTime>`
- Remove `flat()` from `AdaptiveCardRenderer.js`
-  Create `ScreenReaderText` component, which adds `<span>` with text applied to `aria-label` and the innerHTML, which allows for reading the text both in readout mode and scan mode.
- This PR opens up bug #2277
---

-  [ ] Testing Added

### CODE REVIEW

File: packages/component/src/ScreenReaderText.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  height: 1,
+  left: -10000,
+  overflow: 'hidden',
+  position: 'absolute',
+  top: 0,
+  whiteSpace: 'nowrap',
+  width: 1
+});
+
+// const ScreenReaderText = () => false;
```
Comment (compulim) [inline packages/component/src/ScreenReaderText.js]: Removing this line.

File: CHANGELOG.md [hunk 2]
```diff
@@ -63,6 +63,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 -  Fix [#2245](https://github.com/microsoft/BotFramework-WebChat/issues/2245). Fix speech synthesis not working on Safari by priming the engine on the first microphone button click, by [@compulim](https://github.com/compulim) in PR [#2246](https://github.com/microsoft/BotFramework-WebChat/pull/2246)
 -  Fix [#1514](https://github.com/microsoft/BotFramework-WebChat/issues/1514). Added reference grammar ID when using Cognitive Services Speech Services, by [@compulim](https://github.com/compulim) in PR [#2246](https://github.com/microsoft/BotFramework-WebChat/pull/2246)
 -  Fix [#1515](https://github.com/microsoft/BotFramework-WebChat/issues/1515). Added dynamic phrases when using Cognitive Services Speech Services, by [@compulim](https://github.com/compulim) in PR [#2246](https://github.com/microsoft/BotFramework-WebChat/pull/2246)
+-  Fix [#2273](https://github.com/microsoft/BotFramework-WebChat/issues/2273). Add ScreenReaderText component, by [@corinagum](https://github.com/corinagum) in PR [#2278](https://github.com/microsoft/BotFramework-WebChat/pull/2278)
```
Comment (compulim) [inline CHANGELOG.md]:
```suggestion
-  Fix [#2273](https://github.com/microsoft/BotFramework-WebChat/issues/2273). Add `ScreenReaderText` component, by [@corinagum](https://github.com/corinagum) in PR [#2278](https://github.com/microsoft/BotFramework-WebChat/pull/2278)
```

File: packages/component/src/SendBox/ConnectivityStatus.js [hunk 3]
```diff
@@ -62,42 +63,77 @@ const connectConnectivityStatus = (...selectors) =>
...
-          <div className={styleSet.errorNotification}>
-            <ErrorNotificationIcon />
-            {localize('RENDER_ERROR_NOTIFICATION', language)}
-          </div>
-        ) : (
-          connectivityStatus === 'reconnected' || (connectivityStatus === 'connected' && false)
-        )
-      }
+      {() => {
+        if (connectivityStatus === 'connectingslow') {
```
Comment (compulim) [inline packages/component/src/SendBox/ConnectivityStatus.js]: For readability, could you turn this into a `switch` statement?

**Comment by compulim** [comment]
Cross-browser accessibility work is much more harder than cross-browser compatibility work. Good to see how you managed it.

File: packages/component/src/ScreenReaderText.js (hunk 1 above)
Comment (corinagum) [reply packages/component/src/ScreenReaderText.js]: *facepalm*

File: packages/component/src/SendBox/ConnectivityStatus.js (hunk 3 above)
Comment (corinagum) [reply packages/component/src/SendBox/ConnectivityStatus.js]: Question, linting error eslint(no-case-declarations) comes up if I move to switch:
image


Do you still prefer switch statement with this rule disabled at the top of the file?

Comment (corinagum) [reply packages/component/src/SendBox/ConnectivityStatus.js]: lol nvm it needed braces

Comment (compulim) [reply packages/component/src/SendBox/ConnectivityStatus.js]: Oh yea, sorry missed that.

Hate `{}` in switch-case. If you are okay, let's stay with if-then for now.

**Pull request merged**